src/db/Procedures.ts

```diff
@@ -96,4 +103,37 @@ export default class Procedures {
     }
     return res.numRecordsInTable(table);
   }
+
+  public async checkPassFail(
+    procedure: string,
+    parameters: string[],
+    test: ExecutionModelType
+  ) {
+    const tableRows: { [key: string]: number } = {};
+    test.rules.forEach((rule: RuleModelType) => {
+      rule.unitTests.forEach(async (unitTest: UnitTestType) => {
+        if ('tableName' in unitTest) {
```

Comment:
As a cleaner way of handling this I started putting enums in my code on row-operations. I had to add the ones for table so everything compiled so once that's merged to class-transformer you can pull it and update this code

Reply:
Sounds good!

Reply:
These are now in #153 so once that's merged you can pull and update your code